docs/reference/search-application/apis/search-application-search.asciidoc

```diff
@@ -92,28 +100,104 @@ DELETE /index1
 
 ////
 
+[source,console]
```

Comment:
The intent here is to explicitly show the template that the example search application uses. My thought process was that the clearest way to show that was to use the GET call as part of the setup for the example.

We could also show just the template in this doc, but I think that may be confusing because without the GET call, it's unclear how we got the template.

Alternatively, we could link to one of the [search template examples](https://www.elastic.co/guide/en/enterprise-search/current/search-applications-templates.html#search-applications-templates-examples) and say we are using a given template in the API example(s).

Reply:
I think that @leemthompo 's guidance is spot on here. This should be information about the API call and additional context should be in guides. I think it's 100% appropriate to link to guides for more information, in fact that's a pattern I used in 8.10 for [rule_query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-rule-query.html). Nice solution!